python/mxnet/ndarray/random.py

```diff
@@ -220,8 +220,6 @@ def randn(*shape, **kwargs):
     dtype = kwargs.pop('dtype', _Null)
     ctx = kwargs.pop('ctx', None)
     out = kwargs.pop('out', None)
-    assert isinstance(loc, (int, float))
```

Comment:
I guess instead of removing this line it should be
```python
assert isinstance(loc, (int, float, NDArray)) # Whatever the exact type of NDArray is.
```

Reply:
Correct. Makes sense!